### ISSUE TITLE
Impedance Ankle transmission plugin should always work on Cogimon

### ISSUE THREAD

**Issue by FrancescoRuscelli** [ADVRHumanoids/XBotCore#21]
The ImpedanceAnkle plugin, which implement stiffness and damping compliance on the ankle joint, starts only when a move is issued.
Instead, it should always be running independently from the user issuing a move or not, as long as the robot is turned on.

**Comment by alaurenzi** [comment]
I agree. Probably a good solution is to call transmission plugins directly inside the plugin handler loop... Let's discuss about this

**Comment by liesrock** [comment]
Thanks @FrancescoRuscelli, I agree too: it was a bad design choice (documented here https://github.com/ADVRHumanoids/RobotInterfaceXBotRT/issues/3) -> I will keep both the issue updated.

**Comment by alaurenzi** [comment]
This is quite relevant bug, ping!

**Comment by pepperig** [comment]
I think @liesrock  is working on it

**Comment by liesrock** [comment]
Yes, I am on it
